spec/cellular/backends/cool_sms_spec.rb

```diff
@@ -64,4 +69,44 @@
     end
   end
 
+  describe 'parse_response' do
```

Comment:
Could you annotate your describes using '#' and '::' in line with http://betterspecs.org/#describe? That'd be great!